### PR TITLE
QE: Add empty check for the profile card field

### LINKED ISSUE #478: QE: SwiftUI profile card empty space issue
@Luchadores reported:

> When the user doesn’t have any information, we should decrease the space between the name and the link (View profile) as per the design before.

Image

### PR DESCRIPTION
**Pull request by pinarol**
Closes #478 

### Description

Fixes the reported bug. 

Before can be seen here: https://github.com/Automattic/Gravatar-SDK-iOS/issues/478#issuecomment-2402046592

After: 

Screenshot 2024-10-09 at 15 25 55

### Testing Steps

Omit the location from your profile 

Open the QE with the SwiftUI Demo app:
Observe that there's no excess space between the "View profile" button and name

### CODE REVIEW

File: Sources/GravatarUI/SwiftUI/AvatarPicker/AvatarPickerProfileView.swift [hunk 1]
```diff
@@ -29,14 +29,17 @@ struct AvatarPickerProfileView: View {
...
+                        Text(model.location)
+                            .font(.footnote)
+                            .foregroundColor(Color(UIColor.secondaryLabel))
+                    }
                     Button(Localized.viewProfileButtonTitle) {
                         viewProfileAction?()
                     }
                     .font(.footnote)
                     .foregroundColor(Color(UIColor.label))
+                    .padding(.init(top: .DS.Padding.half, leading: 0, bottom: 0, trailing: 0))
```
Comment (pinarol) [inline Sources/GravatarUI/SwiftUI/AvatarPicker/AvatarPickerProfileView.swift:42]: This is not directly related with the bug but according to designs there's a small padding there. So i am using this chance to update this as well.

**Review by etoledom — APPROVED** [review]
Looks good!
